lib/src/main/java/me/onebone/toolbar/CollapsingToolbarScaffold.kt

```diff
@@ -45,6 +49,78 @@ class CollapsingToolbarScaffoldState(
 		get() = offsetYState.value
 
 	internal val offsetYState = mutableStateOf(initialOffsetY)
+
+	// TODO: Maybe should move toolbar expand/collapse methods to CollapsingToolbarState
+	//  but offset expand/collapse leave in CollapsingToolbarScaffoldState?
+
+	// TODO: A strange jump in snap speed is often observed
+	@ExperimentalToolbarApi
+	suspend fun expand(duration: Int = CollapsingToolbarDefaults.ExpandDuration) {
+		val anim = AnimationState(toolbarState.height.toFloat())
+
+		toolbarState.scroll {
+			var prev = anim.value
+			anim.animateTo(toolbarState.maxHeight.toFloat(), tween(duration)) {
+				scrollBy(value - prev)
+				prev = value
+			}
+		}
+	}
+
+	// TODO: A strange jump in snap speed is often observed
+	@ExperimentalToolbarApi
+	suspend fun collapse(duration: Int = CollapsingToolbarDefaults.CollapseDuration) {
+		val anim = AnimationState(toolbarState.height.toFloat())
+
+		toolbarState.scroll {
+			var prev = anim.value
+			anim.animateTo(toolbarState.minHeight.toFloat(), tween(duration)) {
+				scrollBy(value - prev)
+				prev = value
+			}
+		}
+	}
```

Comment:
`expand` and `collapse` method does not do anything with a property of `CollapsingToolbarScaffoldState`, it would be more appropriate to leave it in `CollapsingToolbarState`.